src/modules/unpackfs/main.py

```diff
@@ -258,14 +285,14 @@ def run():
     root_mount_point = globalstorage.value("rootMountPoint")
 
     if not root_mount_point:
-        return ("No mount point for root partition in globalstorage",
+        return ("No mount point for root partition in globalstorage, "
                 "globalstorage does not contain a \"rootMountPoint\" key, "
```

Comment:
This is wrong, you're changing a 2-tuple return (errorstring, explanationstring) to a single string. Leave the comma where it was.